### PR TITLE
Claim module changes

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

File: proto/osmosis/claim/v1beta1/genesis.proto [hunk 1]
```diff
@@ -29,12 +30,12 @@ message UserAction {
 
 // GenesisState defines the claim module's genesis state.
 message GenesisState {
-  // airdrop amount is total amount of airdrop put on claim module's account
-  string airdrop_amount = 1
-    [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int", (gogoproto.nullable) = false];
+  // balance of the claim module's account  
+  cosmos.base.v1beta1.Coin module_account_balance = 1
+      [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coin", (gogoproto.nullable) = false];
```
Comment (ValarDragon) [inline proto/osmosis/claim/v1beta1/genesis.proto:35]: What is `castrepeated`?

File: proto/osmosis/claim/v1beta1/genesis.proto [hunk 2]
```diff
@@ -29,12 +30,12 @@ message UserAction {
...
   // airdrop start time
-  google.protobuf.Timestamp airdrop_start = 2
-    [(gogoproto.stdtime) = true, (gogoproto.nullable) = false, (gogoproto.moretags) = "yaml:\"airdrop_start\""];
+  google.protobuf.Timestamp start_time = 2
```
Comment (ValarDragon) [inline proto/osmosis/claim/v1beta1/genesis.proto:37]: Should this be `airdrop_start_time`, or `chain_start_time`?

File: proto/osmosis/claim/v1beta1/genesis.proto [hunk 3]
```diff
@@ -51,7 +52,7 @@ message GenesisState {
   ];
   // the maximum claimable amount per address
   // [(account_string, max_claimable), (account_string, max_claimable) ... ]
-  repeated cosmos.bank.v1beta1.Balance claimables = 5
+  repeated cosmos.bank.v1beta1.Balance initial_claimables = 5
```
Comment (ValarDragon) [inline proto/osmosis/claim/v1beta1/genesis.proto:55]: Can the comments be updated here to better explain the relation between initial claimable and that its the maximum?

File: x/claim/abci.go [hunk 4]
```diff
@@ -17,6 +17,6 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) {
 	if goneTime > params.DurationUntilDecay+params.DurationOfDecay {
 		// airdrop time passed
 		k.FundRemainingsToCommunity(ctx)
-		k.ClearClaimables(ctx)
+		k.ClearInitialClaimables(ctx)
```
Comment (ValarDragon) [inline x/claim/abci.go:20]: Perhaps `EndAirdrop()`?

Comment (ValarDragon) [inline x/claim/abci.go:20]: I think we should refactor this into a single function called `endAirdrop()`, that then does both of these things internally.

File: x/claim/keeper/claim.go [hunk 5]
```diff
@@ -10,14 +10,20 @@ import (
...
-func (k Keeper) SetModuleAccountBalance(ctx sdk.Context, amount sdk.Int) {
+func (k Keeper) SetModuleAccountBalance(ctx sdk.Context, amount sdk.Coin) {
 	moduleAccAddr := k.accountKeeper.GetModuleAddress(types.ModuleName)
-	k.bankKeeper.SetBalances(ctx, moduleAccAddr, sdk.NewCoins(sdk.NewCoin(k.stakingKeeper.BondDenom(ctx), amount)))
+	k.bankKeeper.SetBalances(ctx, moduleAccAddr, sdk.NewCoins(amount))
 }
 
 // ClearClaimables clear claimable amounts
-func (k Keeper) ClearClaimables(ctx sdk.Context) {
+func (k Keeper) ClearInitialClaimables(ctx sdk.Context) {
```
Comment (ValarDragon) [inline x/claim/keeper/claim.go:26]: lets make this not publicly exposed

File: x/gamm/keeper/multihop_test.go [hunk 6]
```diff
@@ -176,7 +176,7 @@ func (suite *KeeperTestSuite) TestSimpleMultihopSwapExactAmountOut() {
 
 			// Ratio of the oken out should be between the before spot price and after spot price.
 			sp := tokenInAmount.ToDec().Quo(test.param.tokenOut.Amount.ToDec())
-			fmt.Println(spotPriceBefore.String(), spotPriceAfter.String(), sp.String())
+			fmt.Println(spotPriceBefore.String(), spotPriceAfter.String(), sp.String()) // TODO: Remove?
```
Comment (ValarDragon) [inline x/gamm/keeper/multihop_test.go:179]: Yeah, woops. I think I added this for some debugging at some point

Comment (sunnya97) [reply x/gamm/keeper/multihop_test.go:179]:
```suggestion
```

**Review by ValarDragon — COMMENTED** [review]
LGTM

**Pull request merged**